src/ui/ScaleManager.js

```diff
@@ -1,33 +1,61 @@
 import device;
 
-var ScaleManager = exports = Class(function() {
+/** @lends ScaleManager */
+exports = Class(function() {
 
+  /**
+   * @constructs
+   * @param  {number} width
+   * @param  {number} height
+   * @param  {string} [scaleMode=LOCK_HEIGHT] A valid scale mode from {@link ScaleManager.SCALE_MODE}
+   */
   this.init = function(width, height, scaleMode) {
-    this.resize(width, height, scaleMode);
+    /** @type {number} */
```

Comment:
Minor detail here, but we should use `0` instead of `null` for numbers and empty string for string types